frigate/comms/inter_process.py

```diff
@@ -1,16 +1,23 @@
+"""Facilitates communication between processes."""
+
 import multiprocessing as mp
-import queue
+import os
 import threading
-from multiprocessing import Queue
 from multiprocessing.synchronize import Event as MpEvent
 from typing import Callable
 
+import zmq
+
 from frigate.comms.dispatcher import Communicator
+from frigate.const import PORT_INTER_PROCESS_COMM
 
 
 class InterProcessCommunicator(Communicator):
-    def __init__(self, queue: Queue) -> None:
-        self.queue = queue
+    def __init__(self) -> None:
+        INTER_PROCESS_COMM_PORT = os.environ.get("INTER_PROCESS_COMM_PORT") or PORT_INTER_PROCESS_COMM
+        self.context = zmq.Context()
+        self.socket = self.context.socket(zmq.REP)
+        self.socket.bind(f"tcp://*:{INTER_PROCESS_COMM_PORT}")
```

Comment:
I think this should be 127.0.0.1 so it doesn't get exposed externally in host mode.